src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -130,16 +133,22 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
 
   //
   // Exclude the first '/' from the Sub Service
+  // But, only if it starts with a '/'
   //
-  const char* subService = &subServ.c_str()[1];
+  const char* subService = subServ.c_str();
+
+  if (subService[0] == '/')
+  {
+    subService = &subService[1];
+  }
```

Comment:
Another alternative to avoid a "scattered" of wrong service paths should be to use the same group for all them. Maybe the root service path (although the problem in this case is that the metrics get mixed with the ones using correctly the root service path) or use an special keyword.

Not for this PR. For this PR I'd suggest just include the issue in the ".h FIXME list" as something to fix before concluding the functionality. Probably the whole IoTP team should be involved in the discussion.